packages/repositories.config

```diff
@@ -10,6 +10,7 @@
   <repository path="..\src\Common\packages.config" />
   <repository path="..\src\ComputeManagement\packages.config" />
   <repository path="..\src\Configuration.Test\packages.config" />
+  <repository path="..\src\Configuration\packages.config" />
```

Comment:
this should not be added into the repository, I will take a look and get rid of it